scripts/weekly-code-review.sh

```
@@ -0,0 +1,228 @@
+  echo "- _pnpm not available; skipped._" >> "$MD"
+  KNIP_JSON='{"status":"skipped","reason":"pnpm not available"}'
+elif [ ! -f app/knip.json ]; then
+  echo "- _knip config missing; skipped._" >> "$MD"
+  KNIP_JSON='{"status":"skipped","reason":"knip config missing"}'
+else
+  # knip --reporter json prints to stdout; non-zero rc when findings exist.
+  # `--silent` is forwarded to knip (not pnpm) so stdout is raw JSON.
+  (cd app && pnpm exec knip --reporter json) > "$TMP/knip.json" 2> "$TMP/knip.err"
+  knip_rc=$?
+  if [ ! -s "$TMP/knip.json" ]; then
+    echo "- _knip produced no output (rc=$knip_rc); check CI logs._" >> "$MD"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)